### PR TITLE
vdk-jupyter: update npm packages

### PR DESCRIPTION
**Pull request by antoniivanov**
vdk-jupyter build is failing
(https://gitlab.com/vmware-analytics/versatile-data-kit/-/pipelines/1146103715)

It seems there are a lot of outdated packages . As a first step upating them.

### CODE REVIEW

**Review by gageorgiev — APPROVED** [review]
a bunch of packageds have been deprecated it seems, might be worth exploring
```
"node_modules/@babel/plugin-proposal-unicode-property-regex": {
       "version": "7.18.6",
-      "resolved": "https://registry.npmjs.org/@babel/plugin-proposal-unicode-property-regex/-/plugin-proposal-unicode-pro
perty-regex-7.18.6.tgz",
+      "resolved": "http://build-artifactory.eng.vmware.com/api/npm/npm/@babel/plugin-proposal-unicode-property-regex/-/pl
ugin-proposal-unicode-property-regex-7.18.6.tgz",
       "integrity": "sha512-2BShG/d5yoZyXZfVePH91urL5wTG6ASZU9M4o03lKK8u8UW1y08OMttBSOADTcJrnPMpvDXRG3G8fyLh4ovs8w==",
+      "deprecated": "This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-unicode-property-regex instead.",
```

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package-lock.json [hunk 1]
```diff
@@ -464,6 +464,7 @@
       "version": "7.20.7",
       "resolved": "https://registry.npmjs.org/@babel/plugin-proposal-async-generator-functions/-/plugin-proposal-async-generator-functions-7.20.7.tgz",
       "integrity": "sha512-xMbiLsn/8RK7Wq7VeVytytS2L6qE69bXPB10YCmMdDZbKF4okCqY74pI/jJQ/8U0b/F6NrT2+14b8/P9/3AMGA==",
+      "deprecated": "This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.",
```
Comment (DeltaMichael) [inline projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package-lock.json:467]: Please check if we can switch to the recommended plugins for all the babel ones that are deprecated.

**Review by DeltaMichael — APPROVED** [review]

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package-lock.json (hunk 1 above)
Comment (antoniivanov) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package-lock.json:467]: I will do so in separate PR

**Comment by antoniivanov** [comment]
> a bunch of packageds have been deprecated it seems, might be worth exploring
 
 Yes I will do that in separate PR ([link to issue](https://github.com/vmware/versatile-data-kit/issues/3046))

**Pull request merged**